test/built-ins/Array/prototype/concat/is-concat-spreadable-get-order.js

```diff
@@ -0,0 +1,65 @@
+// Copyright (C) 2020 Alexey Shvayka. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-array.prototype.concat
+description: >
+  Symbol.isConcatSpreadable should be looked up once for `this value` and
```

Comment:
JSC looks up `Symbol.isConcatSpreadable` before executing the steps to determine fast or slow path.